### ISSUE TITLE
REFIT

### ISSUE THREAD

**Issue by yhm19971115** [ETSSmartRes/VAE-NILM#9]
Hello, I want to run the REFIT dataset, but there is no corresponding configuration file in the code. Can you give me a solution?

**Comment by ETSSmartRes** [comment]
Hello,

I used the script refit_process.py  from the repository: https://github.com/DLZRMR/seq2subseq/tree/master/preprocess to create the training and test files.

I hope it helps.

Best,

**Comment by yhm19971115** [comment]
Dear author, I tried to process the REFIT data set according to the source code of sub2subseq you provided, but I found that the data of house No. 1 of the REFIT data set was not used as the training set or test set in the data set scene division in your article, and I also found that the data of house No. 11 and House No. 14 were missing. Can you give me some answers? This is very important to my research, I hope to get your reply, thank you.




yuhaimeng9715
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ETSSmartRes/VAE-NILM"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月6日(星期六) 凌晨1:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ETSSmartRes/VAE-NILM] REFIT (Issue #9)





 
Hello,
 
I used the script refit_process.py  from the repository: https://github.com/DLZRMR/seq2subseq/tree/master/preprocess to create the training and test files.
 
I hope it helps.
 
Best,

**Comment by ETSSmartRes** [comment]
Hello,
you can modify the file refit_cfg.py to choose the houses you want for the training and test dataset and when you run refit_process.py, it will look at this file to select which houses you want.

Thanks,
